.github/workflows/trivy.yml

```diff
@@ -0,0 +1,61 @@
+name: "Trivy - Scan for vulnerabilities"
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [main]
+  schedule:
+    # Runs every Monday morning PST
+    - cron: "17 15 * * 1"
+
+jobs:
+  scan-repo:
+    name: Scan repository
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+
+      - name: Run Trivy vulnerability scanner in repo mode
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: "fs"
+          ignore-unfixed: true
+          format: "template"
+          template: "@/contrib/sarif.tpl"
+          output: "trivy-repo-results.sarif"
+          severity: "CRITICAL"
+
+      - name: Upload Trivy scan results to GitHub Security tab
```

Comment:
do you know if CodeQL merges these correctly, since we have two uploads happening in the same PR or build, or does it do something like overwrite one set of results with the other?

asking because I had those issues with Coveralls, so now try to think more carefully :joy:

Reply:
honestly, i have no idea...is it bad to test after merging? 😂

Reply:
nope, sometimes with CI it's a lot of testing in prod lol :joy: good luck!!

Reply:
![image](https://user-images.githubusercontent.com/3806031/116619343-2a982500-a8f5-11eb-9a84-2a1d4934ba75.png)

Reply:
stay reckless, my friends! :cowboy_hat_face: